allensdk/ephys/ephys_features.py

```diff
@@ -544,7 +544,14 @@ def analyze_trough_details(v, t, spike_indexes, peak_indexes, clipped=None, end=
     isi_types = []
 
     update_clipped = []
-    for peak, next_spk in zip(valid_peak_indexes, np.append(valid_spike_indexes[1:], end_index)):
+    
+    # If the last spike was clipped after the peak was achieved, we need to use the last spike's peak index instead of end_index
+    if clipped[-1]==False:
```

Comment:
Please use `if not clipped[-1]`. That is more in keeping with community standards.